cogs/ahk/ahk.py

```diff
@@ -741,9 +742,42 @@ async def solved(self, inter: disnake.AppCmdInter):
         except:  # hdafdsjkhfjdas
             pass
 
+        if (solved_tag in inter.channel.applied_tags):
+            await inter.channel.edit(
```

Comment:
these two branches contain almost exclusively the same code, it should be refactored to rather just change the relevant data inside the conditional logic